library/std/src/sys/pal/windows/mod.rs

```diff
@@ -330,8 +330,13 @@ pub fn dur2timeout(dur: Duration) -> u32 {
 
 /// Use `__fastfail` to abort the process
 ///
-/// This is the same implementation as in libpanic_abort's `__rust_start_panic`. See
-/// that function for more information on `__fastfail`
+/// In Windows 8 and later, this will terminate the process immediately without
+/// running any in-process exception handlers. In earlier versions of Windows,
+/// this sequence of instructions will be treated as an access violation,
+/// terminating the process but without necessarily bypassing all exception
+/// handlers.
```

Comment:
```suggestion
/// In Windows 8 and later, this will terminate the process immediately,
/// bypassing all in-process exception handlers. In earlier versions of Windows,
/// this sequence of instructions will be treated as an access violation,
/// terminating the process but without necessarily bypassing all exception
/// handlers.
```
Let's use the same verb both times, otherwise this is unnecessarily confusing since it uses "without" twice to actually make statements that are opposites of each other.

Reply:
I've rephrased the comment a bit to make it even clearer.